Import/index.mjs

```diff
@@ -0,0 +1,77 @@
+import {Manifest} from "manifesto.js"
+import dbDriver from "../database/driver.mjs"
+
+const database = new dbDriver("mongo")
+
+export class Project {
+  constructor(data) {
+    this.data = data
+  }
+
+  async save() {
+    return database
+      .save({...this.data, "@type": "Project"})
+      .then((savedProject) => {
+       
+        return savedProject
+      }) 
+  }
+
+  static async fetchManifest(projectId) {
+    const url = `https://t-pen.org/TPEN/project/${projectId}`
+    return fetch(url)
+      .then((response) => {
+        return response.json()
+      })
+   }
+
+  static async processManifest(projectId) { 
+    return Project.fetchManifest(projectId).then((manifest) => { 
+       // Process manifest into new Project
+      //Return newProject
+      // UNDER CONSTRUCTION
+
+      let newProject = {}
+
+      const iiifManifest = new Manifest(manifest["@id"]) 
+      const sequences = iiifManifest.getSequences()
+
+      newProject.sequences = sequences
```

Comment:
These are `Pages` for a TPEN3 Project.  A TPEN3 projects does not have the field "sequences".  The Manifest it produces might.  In the scope of TPEN3, each item in the sequence is a Page.  Each item in the sequence may have an AnnotationPage associated with it, which are the Lines of that Page.  You are processing this Manifest for TPEN3 Project metadata, Layers, Pages and Lines